homeassistant/components/tts/__init__.py

```diff
@@ -826,9 +826,8 @@ async def get_tts_data() -> str:
 
             # Save to memory
             if final_extension == "mp3":
-                data = self.write_tags(
-                    filename, data, engine_instance.name, message, language, options
-                )
+                name = engine_instance.name if engine_instance.name is not None else ""
```

Comment:
Why is it ok to use an emtpry string for name here? What does that mean?

Reply:
The name is only used in the album name for the ID3 tag of the `mutagen.File` object.
So the metadata for a tts file generated with this would contain an empty album. According to me looking through the code, this metadata is never used anywhere (and is only present if there is an ID3 tag anyway).